src/Autocomplete/assets/src/controller.ts

```diff
@@ -136,6 +145,9 @@ export default class extends Controller {
             no_results: () => {
                 return `<div class="no-results">${this.noResultsFoundTextValue}</div>`;
             },
+            option_create: (data: TomOption, escapeData: typeof escape_html): string => {
+                return `<div class="create">${this.createOptionTextValue} <strong>${escapeData(data.input)}</strong>&hellip;</div>`;
```

Comment:
It works for french and english (and surely more languages), but is there a lang when you expect the `data.input` to prepend `this.createOptionTextValue`? 

Something like `<div class="create"><strong>${escapeData(data.input)}</strong> ${this.createOptionTextValue} &hellip;</div>`?

Reply:
I think that would be better yes

Reply:
Done :)